templates/terraform/operation.go.erb

```diff
@@ -23,6 +23,10 @@ func (w *<%= product_name -%>OperationWaiter) QueryOp() (interface{}, error) {
 }
 
 func <%= product_name.underscore -%>OperationWaitTime(config *Config, op map[string]interface{}, project, activity string, timeoutMinutes int) error {
+  if op.Name == "" {
```

Comment:
```suggestion
  if val, ok := op["name"]; !ok || val == ""{
```

Reply:
You're telling me that Go Maps return not only the value but an error code because Go loves error codes?

What a language.